### ISSUE TITLE
Support for Python 3.9

### ISSUE THREAD

**Issue by dlangerm** [nalepae/pandarallel#156]
```
 File "/opt/conda/lib/python3.9/site-packages/pandarallel-1.5.2-py3.9.egg/pandarallel/utils/inliner.py", line 32, in wrapper
SystemError: Python version should be 3.{5, 6, 7, 8}
```

Any chance of Python 3.9 support?

**Comment by jorenham** [comment]
https://github.com/nalepae/pandarallel/pull/142

**Comment by nalepae** [comment]
Fixed in `pandarallel v1.5.3` and higher.
